packages/core/entity-list/src/components/ListView/ListView.js

```diff
@@ -62,9 +63,14 @@ const ListView = ({
     tableMinHeight
   ])
 
+  const modifiedFormDefinition = useMemo(
+    () => (formDefinition ? modifyFormDefinition(formDefinition) : formDefinition),
+    [formDefinition, modifyFormDefinition]
+  )
+
   const ActionBar = useMemo(() => {
-    if (formDefinition) {
-      const actionBar = getActionBar(formDefinition)
+    if (modifiedFormDefinition) {
+      const actionBar = getActionBar(modifiedFormDefinition)
```

Comment:
What about on Line 31 ?
Don't we need to use the `modifiedFormDefinition` there as well? I guess for your usecase only changes to the `ActionBar` is need. But the "feature: modifyFormDefintionFn" on the `entity-list` seems to be "not finished" for me.

Reply:
Or we could call the `modifyFormDefinition` function right before we save the `formDefinition` in the state. Then all followup functions / components will use automatically the modified version